org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss

```diff
@@ -386,7 +416,15 @@
             vertical-align: middle;
             border: none;
             padding: 10px 20px;
-            line-height: 1.5em;
+            line-height: 30px;
```

Comment:
Can you replace line-height with extra vertical padding? Line-height solution doesn't looks good if you have a long sentence that will be splitted over two lines